dataflow/src/main/java/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -1057,6 +1074,108 @@ public String toString() {
         }
     }
 
+    /** Receiver takes in binary operations. */
+    public static class BinaryAccess extends Receiver {
+
+        /** The binary node. */
+        protected final BinaryOperationNode node;
```

Comment:
Wouldn't `operation` or something be more meaningful than the very generic `node?